bnd-maven-plugin-parent/bnd-maven-plugin-dependencies/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>biz.aQute.bnd</groupId>
+        <artifactId>bnd-maven-plugin-parent</artifactId>
+        <version>2.5.0-SNAPSHOT</version>
+    </parent>
+    
+    <artifactId>bnd-maven-plugin-dependencies</artifactId>
+    <packaging>pom</packaging>
+    
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>1.7</version>
+                <executions>
+                    <execution>
+                        <phase>initialize</phase>
+                        <configuration>
+                            <target>
+                                <!-- Install the bnd.jar built locally into the local maven repo under the version defined in ${bnd.version} -->
+                                <property name="bnd.jar" value="${basedir}/../../biz.aQute.bnd/generated/biz.aQute.bnd-3.0.0.jar"/>
```

Comment:
the 3.0.0 version in the file name is awkward. I think you should change the value to 

    value="${basedir}/../../dist/bundles/biz.aQute.bnd/biz.aQute.bnd-latest.jar"

This means that the `gradle :dist:releaseNeeded` target (or the default gradle target) need to be build first.

I am also not sure why this is bnd and not bndlib since we want to build the maven plugin against the bndlib we just built with gradle.